timestampvm/vm.go

```diff
@@ -37,7 +37,7 @@ var (
 	Version            = &version.Semantic{
 		Major: 1,
 		Minor: 3,
-		Patch: 1,
+		Patch: 3,
```

Comment:
this jumps 2 patch versions but I think it should align with the GH release version?

Reply:
true, seems we have forgotten to bump it in the previous one.